lib/ui/window/platform_configuration.cc

```diff
@@ -35,7 +35,15 @@ PlatformConfigurationClient::~PlatformConfigurationClient() {}
 
 PlatformConfiguration::PlatformConfiguration(
     PlatformConfigurationClient* client)
-    : client_(client) {}
+    : client_(client) {
+  if (client_->ImplicitViewEnabled()) {
+    // Add PlatformConfiguration's window now, but don't add dart:ui's View
+    // here. The dart:ui needs another way to add the implicit view
+    // synchronously so that the view is available before the main function.
+    // See _implicitViewEnabled in natives.dart.
+    AddWindowRecord(kFlutterImplicitViewId);
```

Comment:
(unrelated to this PR: We should come up with a plan to clean up the Window/View naming in the engine, it's gonna get really confusing once we introduce actual windows. I'll add an item for this to the work tracker.)